2022-10-14 11:12:31,509 INFO    MainThread:31140 [wandb_setup.py:_flush():69] setting env: {'project': 'nyc_airbnb', 'run_group': 'development'}
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_setup.py:_flush():69] setting login settings: {}
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_init.py:_log_setup():336] Logging user logs to /home/smartrue/Dropbox/current_codes/udacity/short-term-rental-prices/src/data_split/wandb/run-20221014_111231-uvd8m3br/logs/debug.log
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_init.py:_log_setup():337] Logging internal logs to /home/smartrue/Dropbox/current_codes/udacity/short-term-rental-prices/src/data_split/wandb/run-20221014_111231-uvd8m3br/logs/debug-internal.log
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_init.py:init():369] calling init triggers
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2022-10-14 11:12:31,510 INFO    MainThread:31140 [wandb_init.py:init():418] starting backend
2022-10-14 11:12:31,510 INFO    MainThread:31140 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-14 11:12:31,521 INFO    MainThread:31140 [backend.py:ensure_launched():132] starting backend process...
2022-10-14 11:12:31,526 INFO    MainThread:31140 [backend.py:ensure_launched():136] started backend process with pid: 31188
2022-10-14 11:12:31,529 INFO    MainThread:31140 [wandb_init.py:init():423] backend started and connected
2022-10-14 11:12:31,531 INFO    MainThread:31140 [wandb_init.py:init():465] updated telemetry
2022-10-14 11:12:31,532 INFO    MainThread:31140 [wandb_init.py:init():484] communicating current version
2022-10-14 11:12:32,417 INFO    MainThread:31140 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-14 11:12:32,417 INFO    MainThread:31140 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-10-14 11:12:32,726 INFO    MainThread:31140 [wandb_init.py:init():522] starting run threads in backend
2022-10-14 11:12:34,233 INFO    MainThread:31140 [wandb_run.py:_console_start():1538] atexit reg
2022-10-14 11:12:34,234 INFO    MainThread:31140 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2022-10-14 11:12:34,234 INFO    MainThread:31140 [wandb_run.py:_redirect():1417] Redirecting console.
2022-10-14 11:12:34,241 INFO    MainThread:31140 [wandb_run.py:_redirect():1473] Redirects installed.
2022-10-14 11:12:34,241 INFO    MainThread:31140 [wandb_init.py:init():547] run started, returning control to user process
2022-10-14 11:12:34,242 INFO    MainThread:31140 [wandb_run.py:_config_callback():787] config_cb None None {'input': 'clean_sample.csv:latest', 'test_size': 0.2, 'random_seed': 42, 'stratify_by': 'none'}
2022-10-14 11:12:45,527 INFO    MainThread:31140 [wandb_run.py:_atexit_cleanup():1508] got exitcode: 0
2022-10-14 11:12:45,527 INFO    MainThread:31140 [wandb_run.py:_restore():1480] restore
2022-10-14 11:12:47,757 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2790244
  total_bytes: 2790244
}

2022-10-14 11:12:48,389 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2790244
  total_bytes: 2790244
}

2022-10-14 11:12:48,762 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 5
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2790244
  total_bytes: 2795550
}

2022-10-14 11:12:49,154 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2790686
  total_bytes: 2795552
}

2022-10-14 11:12:49,256 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,358 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,461 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,563 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,665 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,768 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:49,870 INFO    MainThread:31140 [wandb_run.py:_wait_for_finish():1630] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 2795552
  total_bytes: 2795552
}

2022-10-14 11:12:51,047 INFO    MainThread:31140 [wandb_run.py:_show_files():1852] logging synced files
